aspnetcore/mvc/views/tag-helpers/built-in/anchor-tag-helper.md

```diff
@@ -66,6 +65,33 @@ If no `asp-controller` attribute is specified, the default controller calling th
  
 If the attribute `asp-action` is `Index`, then no action is appended to the URL, leading to the default `Index` method being called. The action specified (or defaulted), must exist in the controller referenced in `asp-controller`.
 
+### asp-page
+
+Use the `asp-page` attribute in an anchor tag to set its URL to point to a specific page. Prefixing the page name with a forward slash "/" creates the URL. The URL in the sample below points to the "Speaker" page in the current directory.
```

Comment:
>Prefixing the page name with a forward slash "/" creates the URL.

Prefixing doesn't create the URL. I think it's more common to use "./" to create relative URLs.

@rynowak  please review.

Reply:
`%2F` is the encoding for "/"
I think it's better to use `./`
Rather than use speaker like the rest of the article, perhaps we should start using exiting samples like https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/page

It doesn't set the URL, it creates a hyperlink.